subsys/storage/flash_map/flash_map_default.c

```diff
@@ -8,54 +8,18 @@
 #include <zephyr.h>
```

Comment:
The way this file was used before is not the same after this PR, this file AFAIK was the hardcoded flash map which could be replaced by a custom one using a Kconfig option. If I understand correctly, it is not needed now, the partitions can now be customised in DTS overlay and there is no need for a user to provide a hardcoded C based default map, am I correct here?

Reply:
Yes you are exactly correct - this file name is confusing so will change it.